Makefile

```diff
@@ -92,7 +95,7 @@ endif
 test-dev: verify test-browser-dev test-webpack
 
 deploy:
-	webpack --bail --config build/deploy/webpack.config.js
+	make build-webpack
```

Comment:
@bjfletcher will omitting `make` or `$(MAKE)` work with other non-make commands. The whole `deploy` command looks like 
```
 deploy:
 	make build-webpack
  	node ./build/deploy/s3.js
  	$(MAKE) build-css-loader
  	$(MAKE) npm-publish
```